plugins/blender-geometry-utilities/README.md

```diff
@@ -0,0 +1,29 @@
+## Blender tools
+
+In order to have your custom character in the metaverse, there is a process and an initial setup, \
+the workflow for this will be using Blender, with a collection of addons and custom tools.
+
+Blender 3.5+ https://www.blender.org/download/
+
+These tools are available below, they are specific versions, you can check the links for different versions \
```

Comment:
I think the links and this text need revising to either point to just product pages with the known working version number next to the link, or links to the specific versions with the product pages provided too.